robottelo/ui/base.py

```diff
@@ -463,3 +420,24 @@ def is_element_visible(self, locator):
             return False
         self.wait_for_ajax()
         return element.is_displayed()
+
+    def click(self, locator, wait_for_ajax=True, timeout=30):
```

Comment:
This is the main part of this Pull Request. The click method was designed to provide better logging when an element can not be found and also automatically wait for ajax calls.